### ISSUE TITLE
stem direction influenced by beams

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#414]
Not a problem, but this seems unusual:  the beam in the following example is controlling the stem direction:

<img width="306" alt="screen shot 2017-01-04 at 15 41 02" src="https://cloud.githubusercontent.com/assets/3487289/21663326/5533614e-d294-11e6-82dc-79e2015fe3a4.png">

It would probably be better if this did not happen, and the algorithm for stem direction should uniform for beamed and unbeamed notes.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt>
                <date>2017-01-04 15:40:48</date>
            </pubStmt>
        </fileDesc>
        <encodingDesc>
            <projectDesc>
                <p>Transcoded from Humdrum with Verovio version 0.9.13-dev-6ee7b51</p>
            </projectDesc>
        </encodingDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-000000109108076">
                        <staffGrp xml:id="m-000000099121035">
                            <staffDef xml:id="staffdef-000000050258138" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4" n="1" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-000000037675474">
                        <measure xml:id="measure-L3" n="1">
                            <staff xml:id="staff-L3F1" n="1">
                                <layer xml:id="layer-L3F1" n="1">
                                    <chord xml:id="chord-L4F1" dots="1" dur="8">
                                        <note xml:id="note-L4F1S1" oct="4" pname="g" accid.ges="n" />
                                        <note xml:id="note-L4F1S2" oct="5" pname="d" accid.ges="n" />
                                    </chord>
                                    <chord xml:id="chord-L5F1" dur="16">
                                        <note xml:id="note-L5F1S1" oct="4" pname="g" accid.ges="n" />
                                        <note xml:id="note-L5F1S2" oct="5" pname="d" accid.ges="n" />
                                    </chord>
                                    <beam xml:id="beam-000000205366459">
                                        <chord xml:id="chord-L6F1" dots="1" dur="8">
                                            <note xml:id="note-L6F1S1" oct="4" pname="g" accid.ges="n" />
                                            <note xml:id="note-L6F1S2" oct="5" pname="d" accid.ges="n" />
                                        </chord>
                                        <chord xml:id="chord-L7F1" dur="16">
                                            <note xml:id="note-L7F1S1" oct="4" pname="g" accid.ges="n" />
                                            <note xml:id="note-L7F1S2" oct="5" pname="d" accid.ges="n" />
                                        </chord>
                                    </beam>
                                    <chord xml:id="chord-L8F1" dur="2">
                                        <note xml:id="note-L8F1S1" oct="4" pname="g" accid.ges="n" />
                                        <note xml:id="note-L8F1S2" oct="5" pname="d" accid.ges="n" />
                                    </chord>
                                    <chord xml:id="chord-L9F1" dur="4">
                                        <note xml:id="note-L9F1S1" oct="4" pname="g" accid.ges="n" />
                                        <note xml:id="note-L9F1S2" oct="5" pname="d" accid.ges="n" />
                                    </chord>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by craigsapp** [comment]
Are you sure that this is the same as #450 ? In that case the stem on the last note in a beam group controls the stem direction of the beam.  In this case the 16th note chord does not have a stem direction.

**Comment by lpugin** [comment]
I think if there is a `stem.dir` in a note within a beam this should set the direction of all the notes in the beam, where ever the note is in the beam. If two (or more) `stem.dir` are given with `up` and `down` values, then this will yield an in-between beam (or double-stemmed beam as they are called by Gould) such as this one https://github.com/music-encoding/music-encoding/issues/391#issuecomment-274619339

I do not think that `beam@place` will work when within a single staff double-stemmed beams. `beam@place` will work with cross-staff cases as long as all notes on the above staff are expected to have a stem down and all notes on the below staff a stem up. If not, then again `@stem.dir` needs to be given explicitly.

In both single staff and cross-staff cases encoding can be non-sense, but I do not think we can avoid it.

**Comment by lpugin** [comment]
Fixed with 7de376debeaa69323358c03f053b1b24595ab0b2

**Comment by craigsapp** [comment]
Looks good:

<img width="575" alt="screen shot 2017-03-06 at 1 24 54 pm" src="https://cloud.githubusercontent.com/assets/3487289/23630528/5273e206-0270-11e7-869b-9c261da24418.png">
